tests/Dashboard/SendMessageTest.php

```diff
@@ -50,7 +50,8 @@ public function can_send_message_on_redis_replication()
                 'data' => json_encode(['data' => 'yes']),
             ])
             ->seeJson([
-                'ok' => true,
+                'exception' => 'Failed to connect to Pusher.',
```

Comment:
yes exactly what I am seeing

Reply:
We should re-analyze what happens in your case. It seems kinda odd. 🤔 This surely is a misconfiguration.

Reply:
three of us worked on this yesterday and could not work out why.
We are using caddy as part of laradock and using a proxy so we do not have to visit eg socket.project.test and project.test to accept the self signed ssl. 

`

project.test:443 {
    root /var/www/project/public
    tls self_signed
    proxy /socket php-worker:6001 {
        without /socket
        transparent
        websocket
    }
    fastcgi / php-fpm:9000 php {
        index index.php
    }
    rewrite {
        if {path} not_starts_with /socket
        to {path} {path}/ /index.php?{query}
    }
}

`

if anything it must have to be the need to pass the path in front in order to use the same host for both normal traffic and websocket traffic